lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -9,39 +14,95 @@ class CategoryMainMenu extends ConsumerStatefulWidget {
         .toList();
 
     return Center(
-        child: GridView.count(
-      primary: false,
-      padding: const EdgeInsets.all(20),
-      crossAxisSpacing: 10,
-      mainAxisSpacing: 10,
-      crossAxisCount: 2,
-      children: gridTile,
-    ));
+      child: Stack(
+        children: [
+          _homeScreenImage(),
+          _homeScreenMenu(homeScreenMenuItems)
+        ],
+      )
+    );
+  }
+
+  Container _homeScreenImage()
+  {
+    return Container(
+      decoration: const BoxDecoration(
+          color: CocktailAppColors.aquaGreen,
+          borderRadius: BorderRadius.vertical(bottom: Radius.circular(90)),
+          image: DecorationImage(image: AssetImage(CocktailAppImages.homeScreenBackground), opacity: 0.68, fit: BoxFit.fill)
+      ),
+      width: 393,
+      height: 440,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: const [
+          Text('COCKTAILS APP', textAlign: TextAlign.center, style: CocktailAppFonts.homeTitle),
+          Text('Be your own Bartender', textAlign: TextAlign.center, style: CocktailAppFonts.homeText)
+        ],
+      ),
+    );
+  }
+
+  Container _homeScreenMenu(List<Widget> homeScreenMenuItems)
+  {
+    return Container(
+      padding: const EdgeInsets.only(top: 335),
+      child: Center(
+          child: GridView.count(
+            primary: false,
+            padding: const EdgeInsets.all(CocktailAppSpacings.homeMenuItemsPaddingSM),
+            crossAxisSpacing: CocktailAppSpacings.homeMenuItemsAxisSpacingSM,
+            mainAxisSpacing: CocktailAppSpacings.homeMenuItemsAxisSpacingSM,
+            crossAxisCount: 2,
+            children: homeScreenMenuItems,
+          )),
+    );
   }
 
-  Widget _buildGridTile({required String withName}) {
+  Widget _buildHomeScreenMenuItems({required CocktailAppHomeMenuItems cocktailAppHomeMenuItem})
+  {
     return GridTile(
       child: Container(
-        padding: const EdgeInsets.all(8),
         decoration: BoxDecoration(
-            border: Border.all(color: Colors.amberAccent, width: 5),
-            color: Colors.amber.shade100,
-            borderRadius: const BorderRadius.all(Radius.circular(20))),
-        child: Center(
-            child: Text(
-          withName,
-          textAlign: TextAlign.center,
-        )),
+          color: CocktailAppColors.white,
+          border: Border.all(color: CocktailAppColors.aquaGreen.withOpacity(0.3), width: 2.0),
+          borderRadius: const BorderRadius.all(Radius.circular(20))
+        ),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children:
+          [
+            Container(
+              height: 100,
+              width: 100,
+              decoration: const BoxDecoration(shape: BoxShape.circle, color: CocktailAppColors.aquaGreen),
+              child: Padding(
+                padding: const EdgeInsets.all(5.0),
```

Comment:
Let's use `CocktailAppSpacings` constant chere
```suggestion
                padding: const EdgeInsets.all(CocktailAppSpacings.spacingXS),
```

Reply:
Suggested changes already pushed. Thank you for your comments and help @keylinwu-dnamic !